frontend/talentsearch/src/js/components/applicantProfile/ExperienceAndSkills.tsx

```diff
@@ -0,0 +1,365 @@
+              experiences={personalExperiences}
+            />
+            <ExperienceByType
+              title={intl.formatMessage({ defaultMessage: "Community" })}
+              icon={<UserGroupIcon style={{ width: "1.5rem" }} />}
+              experiences={communityExperiences}
+            />
+            <ExperienceByType
+              title={intl.formatMessage({ defaultMessage: "Work" })}
+              icon={<BriefcaseIcon style={{ width: "1.5rem" }} />}
+              experiences={workExperiences}
+            />
+            <ExperienceByType
+              title={intl.formatMessage({ defaultMessage: "Education" })}
+              icon={<BookOpenIcon style={{ width: "1.5rem" }} />}
+              experiences={educationExperiences}
+            />
+            <ExperienceByType
+              title={intl.formatMessage({ defaultMessage: "Award" })}
+              icon={<StarIcon style={{ width: "1.5rem" }} />}
+              experiences={awardExperiences}
+            />
+          </Tab>
+          <Tab
+            text={intl.formatMessage({
+              defaultMessage: "By Skills",
+              description:
+                "Tab title for experiences sorted by skills in applicant profile.",
+            })}
+          >
+            <div
+              data-h2-radius="b(s)"
+              data-h2-bg-color="b(lightgray)"
+              data-h2-padding="b(top-bottom, xxs) b(right-left, xs)"
+            >
+              {sortedBySkills.map((skill) => (
+                <SkillAccordion key={skill.id} skill={skill} />
+              ))}
+            </div>
+          </Tab>
+        </TabSet>
+      ) : (
+        <div
+          data-h2-radius="b(s)"
+          data-h2-bg-color="b(lightgray)"
+          data-h2-padding="b(all, m)"
+        >
+          <p data-h2-font-style="b(italic)">
+            {intl.formatMessage({
+              defaultMessage:
+                "There are no experiences on your profile yet. You can add some using the links above.",
+              description:
+                "Message to user when no experiences have been attached to profile.",
+            })}
+          </p>
+        </div>
+      )}
+      <ProfileFormFooter mode="cancelButton" />
+    </ProfileFormWrapper>
+  );
+};
+
+export default ExperienceAndSkills;
+
+export const ExperienceAndSkillsApi: React.FunctionComponent<{
+  applicantId: string;
+}> = ({ applicantId }) => {
+  const intl = useIntl();
+  const [{ data: applicantData, fetching, error }] =
+    useGetAllApplicantExperiencesQuery({ variables: { id: applicantId } });
+
+  if (fetching) return <p>{intl.formatMessage(commonMessages.loadingTitle)}</p>;
+  if (error)
+    return (
+      <p>
+        {intl.formatMessage(commonMessages.loadingError)}
+        {error.message}
+      </p>
+    );
+  return applicantData?.applicant ? (
+    <ExperienceAndSkills
+      experiences={applicantData.applicant.experiences?.filter(notEmpty)}
+    />
```

Comment:
Going off the prior comment, this would all then be unneeded since no API fetch would occur within this component